src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFileSummaryItem.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity.reports;
+
+import java.time.Instant;
+
+public interface ZipFileSummaryItem {
```

Comment:
if there is a future vision of multiple usage - it's ok to have extra interface. or maybe introduce later?